### PR TITLE
bump version to 0.26

### PR DESCRIPTION
**Pull request by apoelstra**

### CODE REVIEW

**Comment by dr-orlovsky** [comment]
Please, please, please one more thing - already two acks, just need to ack rebased version: https://github.com/rust-bitcoin/rust-bitcoin/pull/498

**Comment by dr-orlovsky** [comment]
Discussed potential PRs pending for this release (from IRC):
- [ ] #498 by @dr-orlovsky (ultra-simple, has 1 ack and re-ack by @apoelstra after rebase is required)
- [ ] #496 by @stevenroose (simple; has 1 ack, one more is required)
- [ ] #524 by @sanket1729 (has 1 ack, one more is required)
- [ ] #471 by @dr-orlovsky (not reviewed yet, but finishes epic PSBT refactoring #473)

**Comment by apoelstra** [comment]
#498 I don't think we ever came to a conclusion on the blanket impl of that trait? I'd like an ack from @stevenroose 

I'll take a look at everything else today.

**Comment by apoelstra** [comment]
#524 was pretty controversial, I don't think we're gonna get it in.

**Comment by dr-orlovsky** [comment]
> #498 I don't think we ever came to a conclusion on the blanket impl of that trait? I'd like an ack from @stevenroose

Can pls @stevenroose clarify the situation? From what I am getting from his comments is that he is neither pro or against and asks others to decide

**Comment by apoelstra** [comment]
Rebased. We should be good to go.

**Review by sanket1729 — APPROVED** [review]
Ack d22f3657eb7410723f9f18f805d48eac2b001734 . Awaiting CI

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+- Add [addrv2 support](https://github.com/rust-bitcoin/rust-bitcoin/pull/449)
+- Distinguish [`FilterHeader` and `FilterHash`](https://github.com/rust-bitcoin/rust-bitcoin/pull/454)
+- Add [hash preimage fields](https://github.com/rust-bitcoin/rust-bitcoin/pull/478) to PSBT
+- Detect [write errors for `PublicKey::write_into`](https://github.com/rust-bitcoin/rust-bitcoin/pull/507)
+- impl `Ord` and `PartialOrd` [for `Inventory`](https://github.com/rust-bitcoin/rust-bitcoin/pull/517)
+- Add [binary encoding for BIP32 xkeys](https://github.com/rust-bitcoin/rust-bitcoin/pull/470)
+- Add [Taproot Tagged Hashes](https://github.com/rust-bitcoin/rust-bitcoin/pull/259)
+- Add [`message::MAX_INV_SIZE` constant](https://github.com/rust-bitcoin/rust-bitcoin/pull/516)
+- impl [`ToSocketAddrs` for network addresses](https://github.com/rust-bitcoin/rust-bitcoin/pull/514)
+- Add [new global fields to PSBT](https://github.com/rust-bitcoin/rust-bitcoin/pull/499)
```
Comment (dr-orlovsky) [inline CHANGELOG.md:15]:
```suggestion
- Add [new global fields to PSBT](https://github.com/rust-bitcoin/rust-bitcoin/pull/499)
- [Serde serialization of PSBT data](https://github.com/rust-bitcoin/rust-bitcoin/pull/497)
```

**Review by dr-orlovsky — COMMENTED** [review]
Added one of the things which were missed

File: CHANGELOG.md (hunk 1 above)
Comment (apoelstra) [reply CHANGELOG.md:15]: sure

**Review by dr-orlovsky — APPROVED** [review]

**Review by elichai — APPROVED** [review]

**Review by sgeisler — APPROVED** [review]
ACK 010068ba321268704bc9da9fe311b45b9c0937b6

**Pull request merged**